applications/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1115 @@
+            </doc>
+        </field>
+
+        <field name="start_time" type="NX_DATE_TIME" minOccurs="1" maxOccurs="1">
+            <doc>
+                The starting time(s) of measurement(s) which can be provided in form of a list if multiple measurements are included in the same NXentry.
+            </doc>
+        </field>
+        <field name="end_time" type="NX_DATE_TIME" minOccurs="1" maxOccurs="1">
+            <doc>
+                The end time(s) of measurement(s) which can be provided in form of a list if multiple measurements are included in the same NXentry.
+            </doc>
+        </field>
+
+        <field name="collection_identifier" minOccurs="0" maxOccurs="1">
+            <doc>
+                User or Data Acquisition defined identifier from which
+                the content of this application definition is derived. This can be freely chosen by the user or the instrument scientist and could be, for example, ``05_DA_650_AX_B3P5``, ``SENB-14``, ``Quartz``,....
+            </doc>
+        </field>
+        <field name="collection_description" minOccurs="0" maxOccurs="1">
+            <doc>
+                Brief summary of the collection, including grouping criteria. The information provided in this field can highlight, for example, the measurement setup or information about experimental conditions.
+            </doc>       
+        </field>
+        <field name="processing_type" type="NX_CHAR" minOccurs="1" maxOccurs="1">
+            <doc>
+                Describes the way strain :math:`\varepsilon` can be calculated from the :ref:`center &lt;/NXstress/ENTRY/peaks/center-field&gt;`
+                peak parameter.
+            </doc>
+            <enumeration>
+                <item value="two-theta">
+                    <doc>
+                        :math:`\varepsilon = \large \frac{sin(\mathrm{\theta}_{0})}{sin(\mathrm{\theta})}-1`
+                    </doc>
+                </item>
+                <item value="energy">
+                    <doc>
+                        :math:`\varepsilon = \large \frac{\mathrm{E}_{0}}{\mathrm{E}}-1`
+                    </doc>
+                </item>
+                <item value="d-spacing">
+                    <doc>
+                        :math:`\varepsilon = \large \frac{\mathrm{d}}{\mathrm{d}_{0}}-1`
+                    </doc>
+                </item>
+                <item value="time-of-flight">
+                    <doc>
+                        :math:`\varepsilon = \large \frac{\mathrm{TOF}}{\mathrm{TOF}_{0}}-1`
+                    </doc>
+                </item>
+                <item value="sin2psi">
+                    <doc>
+                        A description of the :math:`\mathrm{\sin}^{2}\psi` method can be found in the literature. Two examples are:
+                        `Fitzpatrick, M. E. et. al.; Determination of residual stresses by X-ray diffraction; National Physical Laboratory, Teddington, 2005`

```

Comment:
Why are these phrase references?

Reply:
Replaced with RST links. I'm assuming that is what you meant.